src/aliceVision/mvsUtils/MultiViewParams.hpp

```diff
@@ -99,9 +100,10 @@ class MultiViewParams
     int g_border = 2;
     bool verbose;
 
-    boost::property_tree::ptree _ini;
+    boost::property_tree::ptree userParams;
```

Comment:
to be consistent should be `_userParams` no?

Reply:
I use the underscore for private member variables. ``userParams`` is public (should be private in the future). I try to reduce the usage of this kind of public member PRs after PRs and transform them as private. In the meantime, I really don't know if I should add or not the underscore.

Reply:
In case of doubt, I really prefer to use the underscore on member variables (and that intentionally public ones shouldn't have it). I find it easier to navigate old code or other people's code when local variables are visually different from member variables.

Reply:
Yes, put underscore on all member variables.
And use accessors to make them public (with the exception of small structs like for x, y, z params).